packages/admin/admin/src/mui/menu/ItemGroup.tsx

```diff
@@ -1,31 +1,60 @@
-import { Box, ComponentsOverrides, Theme, Typography } from "@mui/material";
+import { Box, ComponentsOverrides, Theme, Tooltip, Typography } from "@mui/material";
 import { createStyles, WithStyles, withStyles } from "@mui/styles";
 import * as React from "react";
 
-export type MenuItemGroupClassKey = "root" | "groupTitle";
+export type MenuItemGroupClassKey = "root" | "groupTitleContainer" | "groupTitle";
 
 const styles = (theme: Theme) =>
     createStyles<MenuItemGroupClassKey, MenuItemGroupProps>({
         root: { marginTop: theme.spacing(8) },
         groupTitle: {
             fontWeight: theme.typography.fontWeightBold,
-            fontSize: 14,
+            fontSize: ({ drawerOpen }) => (drawerOpen ? 14 : 12),
+            border: ({ drawerOpen }) => (drawerOpen ? `2px solid ${theme.palette.common.white}` : `2px solid ${theme.palette.grey[100]}`),
+            borderRadius: ({ drawerOpen }) => (drawerOpen ? "initial" : 20),
+            padding: ({ drawerOpen }) => (drawerOpen ? "0" : theme.spacing(2, 4)),
             lineHeight: "20px",
+            color: ({ drawerOpen }) => (drawerOpen ? `${theme.palette.common.black}` : `${theme.palette.grey[300]}`),
+        },
+        groupTitleContainer: {
             borderBottom: `1px solid ${theme.palette.grey[50]}`,
-            padding: theme.spacing(2, 4),
+            display: "flex",
+            justifyContent: ({ drawerOpen }) => (drawerOpen ? "flex-start" : "center"),
+            padding: ({ drawerOpen }) => `${theme.spacing(2)} ${drawerOpen ? theme.spacing(4) : 0}`,
         },
     });
 
 export interface MenuItemGroupProps {
-    title?: string;
+    title: string;
+    drawerOpen?: boolean;
```

Comment:
This should be done via the `MenuContext`. I don't want to have to pass this prop to every MenuItemGroup, because this may be forgotten many times

Reply:
That already changed in #1240

Reply:
That's one problem of using stacked PRs: Things may be concerned in earlier PRs that are already fixed in later PRs. A "correct" Git history would require you to fix this here, where it belongs 🙂

Reply:
You're right. That would be the best.